__tests__/park.test.js

```diff
@@ -20,109 +20,86 @@ async function setupDb() {
     return await setupDb();
   });
 
-  test('Handler - 200 Received list of parks', async () => {
-    const event = {
-      headers: {
-        Authorization: "Bearer " + token
-      },
-      httpMethod: "GET"
-    };
-    expect(await parkGET.handler(event, null)).toMatchObject({
-      body: JSON.stringify(PARKSLIST),
-      headers: {
-        'Access-Control-Allow-Headers': 'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token',
-        'Access-Control-Allow-Methods': 'OPTIONS,GET,POST',
-        'Access-Control-Allow-Origin': '*',
-        'Content-Type': 'application/json'
-      },
-      statusCode: 200
-    });
+  test('Handler - 200 GET fiscal year end', async () => {
+    const obj = await fiscalYearEndGET.handler(
+      {
+        headers: {
+          Authorization: "Bearer " + token
+        },
+        queryStringParameters: {
+          fiscalYearEnd: FISCAL_YEAR_LOCKS[0].sk
+        }
+      }, null);
+    expect(JSON.parse(obj.body)).toMatchObject(FISCAL_YEAR_LOCKS[0]);
   });
 
-  test('Handler - 200 Receive park specific information', async () => {
-    let specificSubAreas = [];
-    for(const area of SUBAREAS) {
-      if (area.pk === "park::0041") {
-        specificSubAreas.push(area);
-      }
-    }
-    const response = await parkGET.handler({
+  test('Handler - 200 GET All fiscal year end objects', async () => {
+    const response = await fiscalYearEndGET.handler({
       headers: {
         Authorization: "Bearer " + token
       },
-      queryStringParameters: {
-        orcs: PARKSLIST[0].sk,
-        subAreaId: specificSubAreas[0].sk
-      }
     }, null);
 
-    const body = JSON.parse(response.body);
-
-    expect(body.data[0].subAreaName).toMatch(body.data[0].subAreaName);
+    expect(response.statusCode).toBe(200);
   });
 
-  test('Handler - 400 GET Bad Request', async () => {
-    const response = await parkGET.handler({
+  test('HandleLock - 200 lock fiscal year', async () => {
+    const response = await fiscalYearEndPOST.lockFiscalYear({
       headers: {
-        Authorization: "Bearer " + token + "invalid"
+        Authorization: "Bearer " + token
       },
       queryStringParameters: {
-        badParam: "oops"
+        fiscalYearEnd: "2018"
       }
     }, null);
 
-    expect(response.statusCode).toBe(400);
+    expect(response.statusCode).toBe(200);
   });
 
-  test('Handler - 400 POST Bad Request', async () => {
-    const response = await parkPOST.handler({
+  test('HandleLock - 200 unlock fiscal year', async () => {
+    const response = await fiscalYearEndPOST.unlockFiscalYear({
       headers: {
-        Authorization: "Bearer " + token + "invalid"
+        Authorization: "Bearer " + token
       },
-      body: JSON.stringify({
-        badParam: "{xxxxxx}"
-      })
+      queryStringParameters: {
+        fiscalYearEnd: "2018"
+      }
     }, null);
 
-    expect(response.statusCode).toBe(400);
+    expect(response.statusCode).toBe(200);
```

Comment:
Instead of just checking for `200 OK`, it would be nice to add actual testing for what the code does - for example, mocking that `putFiscalYear` has been called.  Otherwise we're getting code coverage % without actually validating some of the logic in behind what presents as `200 OK`.

Reply:
Updated the test for better detail.